alembic/versions/a8e9ee2cd4be_add_user_and_updated_date_info_to_.py

```diff
@@ -0,0 +1,77 @@
+"""add_user_and_updated_date_info_to_command_model
+
+Revision ID: a8e9ee2cd4be
+Revises: 00a9ceb38842
+Create Date: 2024-11-27 14:11:59.696512
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'a8e9ee2cd4be'
+down_revision = '00a9ceb38842'
+branch_labels = None
+depends_on = None
+
+# constraint name
+USER_ID_FKEY = 'commandblock_user_id_fk'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('commandblock', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('user_id', sa.Integer(), nullable=True))
+        batch_op.add_column(sa.Column('updated_at', sa.DateTime(), nullable=True))
+        batch_op.alter_column('study_version',
+               existing_type=sa.VARCHAR(length=36),
+               nullable=True,
+               existing_server_default=sa.text("'fake_version'"))
+        batch_op.create_foreign_key(USER_ID_FKEY, 'identities', ['user_id'], ['id'], ondelete='CASCADE')
+
+    with op.batch_alter_table('group_metadata', schema=None) as batch_op:
```

Comment:
Should be removed, same for other tables that have nothing to do with this dev. same for downgrade method